americangut/question/ag_question.py

```diff
@@ -0,0 +1,117 @@
+import inspect
+
+import numpy as np
+
+
+class AgQuestion:
+    """A base object class for handling American Gut Data dictionary entries"""
+    true_values = {'yes', 'y', 'true', 1, 1.0, 'Y', 'YES', 'Yes', True}
```

Comment:
Standardization was complete. The problem comes from trying to remap values here. So, if I remap the values as `'yes'` and `'no'` for one thing, and then decide later that I want to cast the column to bool values, I'd either have to reload the data (which might mean losing other work), or know that `'yes'` and `'no'` are the same as `'true'` and `'false'` for a `bool` column.